Wikipedia/Code/UIViewController+WMFEmptyView.h

```diff
@@ -36,6 +36,10 @@ typedef NS_ENUM(NSUInteger, WMFEmptyViewType) {
 - (void)wmf_applyThemeToEmptyView:(WMFTheme *)theme;
 @property (nonatomic, readonly, nullable) WMFEmptyView *wmf_emptyView;
 
++ (nullable WMFEmptyView *)emptyViewOfType:(WMFEmptyViewType)type action:(nullable SEL)action theme:(WMFTheme *)theme frame:(CGRect)frame;
```

Comment:
looks like this one is not used anywhere

Reply:
@nambatee removed